src/pages/RegisterPage2.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <main class="register-container">
+    <MainTitle
+      text="Great!"
+      subText="Now your name"
+      class="register-container__title"
+    />
+
+    <FormName />
+
+  </main>
+</template>
+
+<script>
+import MainTitle from "../components/MainTitle.vue";
+import FormName from "../components/FormName.vue";
+
+export default {
+  name: "RegisterPage2",
+  components: { MainTitle, FormName },
+};
+</script>
+
+<style lang="scss">
+@media (min-width: 0px) {
```

Comment:
Is this `min-width: 0px` needed here? If those styles should apply at every screen resolution then I think it's redundant

Reply:
Thanks for all hints - I implemented them to the code.
I will work later on prettier settings.